src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -21,12 +21,21 @@
 import json
 import logging
 import os
+import subprocess
 import time
 import traceback
+from parkbenchcommon import broadcastconsumer
 import gpgkeyverifier
 import gpgmailbuilder
-import mailsender
 
+# The number of seconds to wait after a broadcast to get another broadcast.
+BROADCAST_NETCHECK_GATEWAY_CHANGED_DELAY = 5
+BROADCAST_NETCHECK_GATEWAY_CHANGED_NAME = 'gateway-changed'
+
+class SendmailError(Exception):
+    """This exception is raised when sendmail returns an error code upon attempting to queue
```

Comment:
This isn't an error code in the traditional sense. It is an error code indicating sendmail failed. Maybe update the message to reflect this.